libs/community/tests/integration_tests/test_googlesearch_api.py

```diff
@@ -1,32 +1,41 @@
 """Integration test for Google Search API Wrapper."""
 
-import pytest
+import os
 
 from langchain_google_community.search import GoogleSearchAPIWrapper
 
 
-@pytest.mark.skip(reason="CI/CD not ready.")
 def test_call() -> None:
```

Comment:
should we mark it as extended (and add a corresponding flag to pytest conf)?

Reply:
done